### ISSUE TITLE
Virtual Pad

### ISSUE THREAD

**Issue by tonebiclas** [aldostools/webMAN-MOD#304]
Hello, just discovered virtualpad routes recently and made an small local web app to control my ps3 with the keyboard, could play tomb raider 1 with it... much fun that game helped me a bit to understand the combination of keys and its playable, but...
Sometimes the requests takes some time or it cause alot of actions when one is sent (similar to hold the key) example "up" repeat 3 times or more or actually hold like if its on "hold_up" but it is just "up", and im just sending one "up" its weird. Can that routes be improved somehow?

Also the virtualpad isn't working on ps3 games or im missing something?

**Comment by aldostools** [comment]
Check http://pad.aldostools.org

It has 2 modes: one for xmb and another for games. That's the best I could do.

**Comment by tonebiclas** [comment]
Good.
This isn't triggering keys inside ps3 games aswell, only psbtn seems to work there. PS1 games seems to work, the few i tested. Do you know why?

**Comment by aldostools** [comment]
Did you try switching from "Normal Mode" to "Game Mode"? (click on the label below the IP address)

The last time I tested it it worked with PS3 games. Although it cannot be used on action games, due the lag.

**Comment by tonebiclas** [comment]
Yes. I have game mode when im clicking.
Can you test and see if works? Im not using cfw its hen (if that matters, or type of game?)

**Comment by aldostools** [comment]
No, I'm at work and I won't be able to use the console in the next days.

It doesn't matter if it's HEN or CFW. The type of game matters, because they poll the game pad in different ways. Some games require fast response.

**Comment by Joonie86** [comment]
No further feedback
